style.css

```diff
@@ -0,0 +1,237 @@
+  font-weight: 600;
+  font-style: normal;
+  line-height: 16px;
+  color: #344563;
+}
+
+.back-end {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.access-project {
+  width: 295px;
+  height: 76px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 10px;
+}
+
+.programming-lang {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 70%;
+  margin-top: 10px;
+}
+
+.lang {
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  color: #6070ff;
+  letter-spacing: 0.03em;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.project-btn {
+  display: flex;
+  margin-top: 15px;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  width: 128px;
+}
+
+button {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  border: none;
+  background-color: white;
+}
+
+.aboutMe-container {
+  width: 100%;
+  margin-top: 20%;
+  background-color: #fff;
+  padding: 16px;
+  border-top-right-radius: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.me {
+  width: 32px;
+  height: 101px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  margin-top: 20%;
+}
+
+.my-qualities {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 5%;
+}
+
+.lets-connect {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  margin-top: 5px;
+}
+
+.social-media {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 8%;
+  width: 60%;
+}
+
+.project-btn2 {
+  margin-top: 11px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  width: 165px;
+}
+
+.language-title {
+  margin-top: 11%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.language {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+}
+
+.down-arrow {
+  border: 2px solid #acb7c3;
+}
+
+.languages {
+  display: grid;
+  grid-template-columns: 23% 23%;
+  align-items: center;
+  margin-top: 5%;
+  background: #f7f7f9;
+  border-radius: 8px;
+  padding: 5px;
+  width: 303px;
+}
+
+.framework {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 270px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  margin-top: 5%;
+  margin-bottom: 5%;
+}
```

Comment:
Kindly make sure your sections match the [Figma Template :art: ] as stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) (https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) 
**Your design:**
![image](https://user-images.githubusercontent.com/92395484/192062592-80d7c02a-3146-454b-be70-4f3d108796b1.png)

**Figma Template:**
![image](https://user-images.githubusercontent.com/92395484/192062664-7217204a-a1f5-4dcb-a81c-5957f04acb15.png)


**Note:** This includes background colours, position and alignment, margin and paddings :eyes:

In this case attention to detail is highly encouraged :raised_hands:

Reply:
Hello, @karlavdelgadof Thank you for the feedback,
I'll add the required changes to my project